src/object/srv_ec_aggregate.c

```diff
@@ -1528,11 +1529,15 @@ agg_process_holes_ult(void *arg)
 	ec_rep_in->er_map_ver =
 		agg_param->ap_pool_info.api_pool->sp_map_version;
 	entry->ae_sgl.sg_nr_out = 1;
-	rc = crt_bulk_create(dss_get_module_info()->dmi_ctx, &entry->ae_sgl,
-			     CRT_BULK_RW, &ec_rep_in->er_bulk);
-	if (rc) {
-		D_ERROR("crt_bulk_create returned: "DF_RC"\n", DP_RC(rc));
-		goto out;
+	if (ext_cnt) {
+		rc = crt_bulk_create(dss_get_module_info()->dmi_ctx,
+				     &entry->ae_sgl, CRT_BULK_RW,
+				     &ec_rep_in->er_bulk);
```

Comment:
need to free the bulk handle, but i had fixed it in my PR https://github.com/daos-stack/daos/pull/5109

Reply:
Fixed.